src/runtime/cpu/kernels/kernels.cpp

```diff
@@ -46,3 +46,27 @@ void deepworks::CPUReLUBackward(ConstMatrix dx, ConstMatrix output, Matrix grad_
 
     grad_output = (output.array() > 0.0).select(dx, 0.0);
 }
+
+void deepworks::CPULog(ConstMatrix X, Matrix LogX) {
+    LogX.array() = X.array().log();
+}
+
+std::vector<int> deepworks::MatchTargetTo1dMatrix(ConstVector target, int batch_size, int n_classes) {
```

Comment:
It used only in this file, I would make it static

Reply:
Oh, I see it's a part of `kernels`, should it be used by someone else ? If yes, not changes are required